src/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -115,6 +115,10 @@ public partial struct ImmutableArray<T> : System.Collections.Generic.ICollection
         bool System.Collections.IList.IsFixedSize { get { throw null; } }
         bool System.Collections.IList.IsReadOnly { get { throw null; } }
         object System.Collections.IList.this[int index] { get { throw null; } set { } }
+#if !NETSTANDARD10
+        public System.ReadOnlySpan<T> AsSpan() { throw null; }
+        public System.ReadOnlyMemory<T> AsMemory() { throw null; }
```

Comment:
nit: Was this auto-generated using `/t:GenerateReferenceSource`? I believe they need to be re-ordered (alphabetically).